Sources_v3/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -58,7 +58,7 @@ class ChannelListController_Tests: StressTestCase {
         XCTAssertEqual(controller.state, .localDataFetched)
         
         // Simulate successfull network call.
-        env.channelQueryUpdater?.update_completion?(nil)
+        env.channelListUpdater?.update_completion?(nil)
```

Comment:
This is probably not correct?? Or I don't understand what's happening here in the "Add markAllRead functionality and tests" commit

Reply:
I've mixed the renaming commit with add markAllRead commit... Since I renamed `ChannelListQueryUpdater -> ChannelListUpdater` I've also renamed related properties in tests

Reply:
I got confused because the original naming here was clearly wrong, too. Sounds like the "channel query" updater, not "channel list query"